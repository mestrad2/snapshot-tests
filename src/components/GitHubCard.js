import React from "react"
import Card from "react-bootstrap/Card"
import MyPic from './self_image.jpg'

function GitHubCard() {
  return (
      <div className="card">
        <Card style={{ width: "18rem"}}>
        <Card.Img variant="top" src={MyPic} />
        <Card.Body>
            <Card.Title>Manuel Estrada / mestrad2</Card.Title>
            <Card.Text>
            I currently work for Wells Fargo and I enjoy gaming as a hobby.
            </Card.Text>
        </Card.Body>
        </Card>
      </div>
  );
}

export default GitHubCard