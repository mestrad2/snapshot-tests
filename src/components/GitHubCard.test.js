import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
    const pic = renderer
    .create(<GitHubCard />)
    .toJSON()
    expect(pic).toMatchSnapshot()
})