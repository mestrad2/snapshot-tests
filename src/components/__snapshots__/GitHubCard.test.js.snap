// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="self_image.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Manuel Estrada / mestrad2
      </div>
      <p
        className="card-text"
      >
        I currently work for Wells Fargo and I enjoy gaming as a hobby.
      </p>
    </div>
  </div>
</div>
`;
